tests/periph_wdt/tests/01-run.py

```diff
@@ -7,13 +7,27 @@
 # directory for more details.
 
 import sys
+import os
 import pexpect
 import time
+import subprocess
 from testrunner import run
 
+# nucleo-l152re is the default platform for this test
+BOARD = os.environ.get("BOARD", "nucleo-l152re")
+CPU = subprocess.check_output([
+    'make',
+    'info-debug-variable-CPU',
+    'BOARD={}'.format(BOARD),
+    '--no-print-directory']).strip()
```

Comment:
looks like Travis doesn't like this :confused:

Reply:
Lets just change the variable name, it does not need to me `CPU`, can be `BOARD_CPU` and I think travis would not complain.